common/src/Model/Brush.cpp

```diff
@@ -894,7 +916,15 @@ namespace TrenchBroom {
         Vec3 Brush::splitFace(const BBox3& worldBounds, const Polygon3& facePosition, const Vec3& delta) {
             return addVertex(worldBounds, facePosition.center() + delta)->position();
         }
-       
+        
+        static std::pair<bool, BrushGeometry> rejectVertexMove() {
```

Comment:
I would prefer to use a private class `CanMoveVerticesResult` and put the static methods there. I prefer to create dedicated classes when I have to return multiple values from the same methods.